client/src/constants/backend_constants.js

```diff
@@ -1,11 +1,89 @@
-import deepFreeze from 'deep-freeze';
 import generatedBackendConstants from '@/../generated/backend_constants.json';
+import forms from '@/constants/forms.js';
+import deepFreeze from 'deep-freeze';
 
 /**
  * Note that these are generated by running `npm run setup` first.
  *
  * @typedef HMModel
```

Comment:
if HMModel is no longer being referenced anywhere in the application, this type can likely be removed.

Reply:
Removed.